### PR TITLE
Implement Create Road Station

### PR DESCRIPTION
**Pull request by duncanspumpkin**

### CODE REVIEW

**Comment by duncanspumpkin** [comment]
No divergence on both my ai saves but they don't use road stations too much.

**Review by AaronVanGeffen — APPROVED** [review]
Road stations can be placed normally and appear to be picked up properly by buses, so it's a 👍 from me.

**Comment by duncanspumpkin** [comment]
No divergences on replays.

**Review by AaronVanGeffen — APPROVED** [review]

File: src/OpenLoco/src/GameCommands/Road/CreateRoadStation.cpp [hunk 1]
```diff
@@ -0,0 +1,685 @@
...
+        regs.bp = tad;
+        regs.bh = roadObjectId;
+        call(0x004900B8, regs);
+        NearbyStation result{};
+        result.id = static_cast<StationId>(regs.bx);
+        result.isPhysicallyAttached = regs.eax & (1U << 31);
+        return result;
+    }
+
+    static bool sub_48FEF4(StationId id, World::Pos3 pos)
```
Comment (duncanspumpkin) [inline src/OpenLoco/src/GameCommands/Road/CreateRoadStation.cpp:49]: #2306

**Pull request merged**